trunner/harness/psh.py

```diff
@@ -72,7 +72,14 @@ def __call__(self, result: TestResult) -> TestResult:
         if self.cmd is not None:
             self.dut.send(self.cmd + "\n")
             try:
-                self.dut.expect(f"{self.cmd}(\r+)\n")
+                if len(self.cmd) <= 68:
```

Comment:
Maybe it would be worth adding a comment explaining why this 'if' statement is being needed, and why the number 68

Reply:
I agree on that, we don't know why just 68

Reply:
It should also be in separate commit